package.json

```diff
@@ -9,18 +9,20 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@types/react-router-dom": "^5.3.3",
+    "next": "14.2.2",
     "react": "^18",
     "react-dom": "^18",
-    "next": "14.2.2"
+    "react-router-dom": "^6.23.1"
```

Comment:
Why do we need a new dependency if we alredy do routing without

Reply:
I have now deleted it